### PR TITLE
Implemented taxes import feature

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Implemented taxes import feature

### CODE REVIEW

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php [hunk 1]
```diff
@@ -286,44 +283,34 @@ public function import()
...
                                     $this->taxRateRepository->create($uploadData);
                                 }
                             }
                         }
 
-                        session()->flash('success', trans('admin::app.export.upload-success', ['name' => 'Tax Rate']));
+                        return new JsonResponse([
+                            'message' => trans('admin::app.export.upload-success', ['name' => 'Tax Rate']),
+                        ]);
                     }
```
Comment (suraj-webkul) [inline packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php:303]: Use proper translation remove hard coded text

**Review by suraj-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php (hunk 1 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php]: fixed

**Review by suraj-webkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php [hunk 2]
```diff
@@ -4,23 +4,139 @@
...
+                </button>
+
+                <!-- Modal Form -->
+                <x-admin::form
+                    v-slot="{ meta, errors, handleSubmit }"
+                    as="div"
+                >
+                    <form
+                        @submit="handleSubmit($event, create)"
+                    >
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php:57]: Remove the separations. Do it in a single line.

File: packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php [hunk 3]
```diff
@@ -4,23 +4,139 @@
...
+                        resetForm,
+                        setErrors
+                    }) {
+                        const formData = new FormData();
+
+                        const fileInput = this.$refs.importFile;
+
+                        if (fileInput.files.length > 0) {
+                            formData.append('file', fileInput.files[0]);
+                        }
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php:120]: Remove the conditional opeartor

**Review by amit-webkul — CHANGES_REQUESTED** [review]

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 
Please resolve the conflicts and unresolved conversations.

File: packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php (hunk 2 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php]: Fixed

File: packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php (hunk 3 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php]: fixed

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php [hunk 4]
```diff
@@ -176,28 +176,25 @@ public function destroy($id): JsonResponse
...
-                        if (is_null($uploadData['state'])) {
-                            $uploadData['state'] = '';
-                        }
+                        $uploadData['state'] = $uploadData['state'] ?? '';
 
-                        if (
-                            ! is_null($uploadData['zip_from'])
-                            && ! is_null($uploadData['zip_to'])
-                        ) {
+                        if (! is_null($uploadData['zip_from']) && ! is_null($uploadData['zip_to'])) {
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php:197]: Break the properties into multiple lines.

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php [hunk 5]
```diff
@@ -286,44 +283,34 @@ public function import()
...
-                                }
+                                $uploadData['state'] = $uploadData['state'] ?? '';
 
-                                if (
-                                    ! is_null($uploadData['zip_from'])
-                                    && ! is_null($uploadData['zip_to'])
-                                ) {
-                                    $uploadData['is_zip'] = 1;
-                                    $uploadData['zip_code'] = null;
+                                if (! is_null($uploadData['zip_from']) && ! is_null($uploadData['zip_to'])) {
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php:288]: Same Here.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php (hunk 5 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php]: fixed

File: packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php (hunk 4 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php]: fixed

**Review by amit-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**